internal/cmd/kafka/command_topic_create_onprem.go

```diff
@@ -63,10 +63,10 @@ func (c *authenticatedTopicCommand) createOnPrem(cmd *cobra.Command, args []stri
 		return err
 	}
 
-	return CreateTopicWithRestClient(cmd, restClient, restContext, topicName, clusterId)
+	return CreateTopicWithRESTClient(cmd, restClient, restContext, topicName, clusterId)
```

Comment:
I think we decided to format functions this way- this helps with readability in certain cases: `TestIamRbac` is easier to read than `TestIAMRBAC` for instance.
```suggestion
	return CreateTopicWithRestClient(cmd, restClient, restContext, topicName, clusterId)
```